<launch>
    <arg name="sonar_address" default="/home/schvarcz/Desktop/sonar_guerledan/Oct_13_2017_110048.son" />
    <arg name="color_map_file" default="$(find blueview_ros)/colormaps/jet.cmap" />

    <node pkg="blueview_ros" name="blueview_ros_image" type="blueview_ros_node" output="screen">
        <param name="address" type="string" value="$(arg sonar_address)" /> <!-- if net, ip address : if file, path -->
        <param name="net_or_file" type="string" value="file" /> <!-- file or net -->
        <param name="enable_range_data" type="bool" value="true" />
        <param name="enable_gray_image" type="bool" value="true" />
        <param name="enable_colored_image" type="bool" value="true" />
        <param name="enable_point_cloud" type="bool" value="true" />
        <param name="color_map" type="string" value="$(arg color_map_file)" />
        <param name="tick_rate" type="double" value="10.0" />
        <param name="max_dist" type="double" value="40" />
        <param name="min_dist" type="double" value="0" />
        <param name="base_link_name" type="string" value="base_link" />
        <param name="time_diff" value="0"/>
        <param name="first_time" value="0"/>
        <!--<param name="time_diff" value="7200"/>-->
        <!--<param name="first_time" value="39596"/>-->
    </node>

    <!--<node name="rviz" pkg="rviz" type="rviz" args="-d $(find blueview_ros)/config/rviz.rviz" />-->
</launch>
